doc/nrf/releases_and_maturity/releases/release-notes-3.0.0.rst

```diff
@@ -0,0 +1,1441 @@
+
+Google Fast Pair integration
+----------------------------
+
+* Added:
+
+  * Instructions on how to provision the Fast Pair data onto devices without the :ref:`partition_manager` support, specifically for the :ref:`zephyr:nrf54h20dk_nrf54h20`.
+  * Information on how to support the firmware update intent feature on the Android platform.
+    Expanded the documentation for the Fast Pair devices with the FMDN extension, which requires additional steps to support this feature.
+
+* Updated:
+
+  * The :ref:`ug_bt_fast_pair_provisioning_register_hex_generation` section that describes how to generate the hex file with the Fast Pair provisioning data.
+  * The :ref:`ug_bt_fast_pair_prerequisite_ops_kconfig` section to align it with recent changes in the sysbuild configuration for Fast Pair.
+  * The :ref:`ug_bt_fast_pair_gatt_service_fmdn_info_callbacks_provisioning_state` section with changes to the FMDN API elements that are used for tracking of the FMDN provisioning state.
+
+Memfault integration
+--------------------
+
+* Added a new feature to automatically post coredumps to Memfault when network connectivity is available.
+  To enable this feature, set the :kconfig:option:`CONFIG_MEMFAULT_NCS_POST_COREDUMP_ON_NETWORK_CONNECTED` Kconfig option to ``y``.
+  Only supported for nRF91 Series devices.
+
+* Added a new feature to automatically capture and upload modem traces to Memfault with coredumps upon a crash.
+  To enable this feature, set the :kconfig:option:`CONFIG_MEMFAULT_NCS_POST_MODEM_TRACE_ON_COREDUMP` Kconfig option to ``y``.
+  Only supported for nRF91 Series devices.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+* Added :ref:`soft_peripherals`.
+* Removed the Zboss documentation.
+  It is now available in separate `Zigbee R22`_ and `Zigbee R23`_ add-on repositories (depending on the device you are working with).
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a2bc982b3379d51fefda3e17a6a067342dce1a8b``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Fixed an issue where an unusable secondary slot was cleared three times instead of once during cleanup.
+* Added keys revocation scheme support for nRF54L SoCs, see ``CONFIG_BOOT_KEYS_REVOCATION`` MCUboot Kconfig option.
+* Improved time performance of firmware update by usage of best RRAM write operation buffering on nRF54L SoCs.
+* Introduced improved swap algorithm: swap-using-offset, see ``CONFIG_BOOT_SWAP_USING_OFFSET`` MCUboot Kconfig option.
+* Image compression support has been brought to production quality.
+
```

Comment:
Experimental support for encrypted DFU with ECIES x25519 encryption using MCUboot for nRF54l.

Reply:
Added